src/main/java/com/hedvig/contract/validator/ContractProductValidator.kt

```diff
@@ -49,7 +49,7 @@ class ContractProductValidator(
     }
 
     try {
-      val currentOrLatestProduct = insuranceService.getCurrentOrLastInsurance(memberId).orElseThrow()
+      val currentOrLatestProduct = insuranceService.getCurrentOrLastInsurance(memberId, true).orElseThrow()
```

Comment:
We never compare anything with the member so this can be false
```suggestion
      val currentOrLatestProduct = insuranceService.getCurrentOrLastInsurance(memberId, false).orElseThrow()
```